src/Tizen.Data.Tdbc/Tizen.Data.Tdbc/RecordChangedEventArgs.cs

```diff
@@ -0,0 +1,72 @@
+﻿/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.Data.Tdbc
+{
+    /// <summary>
+    /// RecordChangedEventArgs class. This class is an event arguments of the RecordChanged events.
+    /// </summary>
+    /// <since_tizen> 11 </since_tizen>
+    public class RecordChangedEventArgs : EventArgs
+    {
+        private readonly OperationType _operationType;
+        private readonly string _database;
+        private readonly string _table;
+        private readonly long _rowId;
+
+        /// <summary>
+        /// Creates and initializes a new instance of type of the RecordChangedEventArgs class.
+        /// </summary>
+        /// <param name="operationType">The operation type of the changed record.</param>
+        /// <param name="database">The database of the changed record.</param>
+        /// <param name="table">The table of the changed record.</param>
+        /// <param name="rowId">The rowId of the changed record.</param>
+        /// <since_tizen> 11 </since_tizen>
+        public RecordChangedEventArgs(OperationType operationType, string database, string table, long rowId)
```

Comment:
The instance of this class should be generated by drivers. When i changed this constructor to internal, it can't be accessed by driver. So i added "[EditorBrowsable(EditorBrowsableState.Never)]" instead. Please tell me if there is another way.

Reply:
ok, that makes sense